README.md

```diff
@@ -51,3 +51,16 @@ using the following command:
 ```shell
 $ ENABLE_WEBHOOKS=false make run install
 ```
+
+## Metrics
+
+Apart from the [metrics provided by controller-runtime](https://book.kubebuilder.io/reference/metrics-reference.html)
+by default, this operator exports the following custom metrics:
+
+| Metric name                      | Type      | Description                                                                |
+|----------------------------------|-----------|----------------------------------------------------------------------------|
+| release_attempt_concurrent_total | Gauge     | Total number of concurrent releases.                                       |
+| release_attempt_duration_seconds | Histogram | Total duration of the release from the moment the PipelineRun was created. |
```

Comment:
As talked about, the description should be updated to be more clear

Reply:
Changed to `Releases duration from the moment the release PipelineRun was created`. Not sure it's better though. Suggestions accepted.

Reply:
`Total time for all releases to change their state to finished, starting from the moment their release PipelineRun was created` ? That could probably be better English... :man_shrugging: 

And to make sure I remember this correctly, it is state running to state completed, so the processing/validation time in the operator is not included?